scylla_docker.py

```diff
@@ -159,7 +160,7 @@ def get_stress_results(stress_out):
                 try:
                     res = line.split(':')
                     key = res[0].strip()
-                    val = res[1].split()[0].strip() if key != 'Total operation time' else\
+                    val = res[1].split()[0].strip().replace(',', '') if key != 'Total operation time' else\
```

Comment:
Let's split this change to another patch :-)

Reply:
Split. Remove from this commit